niceml/dlframeworks/tensorflow/callbacks/csvlogger.py

```diff
@@ -26,13 +26,37 @@ class CSVLogger(Callback):
     def __init__(
         self, experiment_context: ExperimentContext, filename: str = "train_logs.csv"
     ):
+        """Callback that streams epoch results to a CSV file.
+
+        Supports all values that can be represented as a string,
+        including 1D iterables such as np.ndarray.
+
+        # Example
+        ```python
+        csv_logger = CSVLogger(exp_context=exp_context)
+        model.fit(X_train, Y_train, callbacks=[csv_logger])
+        ```
+
+        # Arguments
+            exp_context: An instance of ExperimentContext which is used to write csv files
+            separator: string used to separate elements in the CSV file.
```

Comment:
Check if the 'separator' is still part of the CSVLogger function.

Reply:
It's not. I've removed it from the dosctring